### PR TITLE
[stable10] extended theme service

### PR DESCRIPTION
**Pull request by PVince81**
Backport of https://github.com/owncloud/core/pull/27746/ to stable10

I haven't retested this...

@DeepDiver1975

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
Let me test this ....

File: lib/public/Theme/ITheme.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ */
+namespace OCP\Theme;
+
+/**
+ * The representation of a Theme.
+ * @package OCP\Theme
+ * @since 10.0.1
```
Comment (DeepDiver1975) [inline lib/public/Theme/ITheme.php]: 10.0.3

File: lib/public/Theme/IThemeService.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ */
+namespace OCP\Theme;
+
+/**
+ * @package OCP\Theme
+ * @since 10.0.1
```
Comment (DeepDiver1975) [inline lib/public/Theme/IThemeService.php]: 10.0.3

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: lib/public/Theme/ITheme.php (hunk 1 above)
Comment (PVince81) [inline lib/public/Theme/ITheme.php]: argh, I should have reread through the original PR... so old. I'll do a follow up PR for master

Comment (PVince81) [inline lib/public/Theme/ITheme.php]: fixed

Comment (PVince81) [inline lib/public/Theme/ITheme.php]: https://github.com/owncloud/core/pull/28650 for master

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by PVince81** [comment]
Tests passed but were not reported: https://jenkins.owncloud.org/job/owncloud-core/job/core/job/PR-28647/

**Pull request merged**